src/app/activity/bounty/Publish.ts

```diff
@@ -1,35 +1,45 @@
-import { TextChannel, Message, MessageEmbedOptions } from 'discord.js'
-import Log, { LogUtils } from '../../utils/Log';
+import { TextChannel, Message } from 'discord.js'
+import Log from '../../utils/Log';
 import mongo, { Db, UpdateWriteOpResult } from 'mongodb';
 import MongoDbUtils from '../../utils/MongoDbUtils';
 import { BountyCollection } from '../../types/bounty/BountyCollection';
 import { CustomerCollection } from '../../types/bounty/CustomerCollection';
 import DiscordUtils from '../../utils/DiscordUtils';
 import BountyUtils from '../../utils/BountyUtils';
-import RuntimeError from '../../errors/RuntimeError';
 import { PublishRequest } from '../../requests/PublishRequest';
 import { BountyStatus } from '../../constants/bountyStatus';
 import { Activities } from '../../constants/activities';
 import { Clients } from '../../constants/clients';
+import Bounty from '../../commands/bounty/Bounty';
 
 export const publishBounty = async (publishRequest: PublishRequest): Promise<any> => {
 	Log.debug('In Publish activity');
 
-	Log.info(`starting to finalize bounty: ${publishRequest.bountyId} from guild: ${publishRequest.guildId}`);
 	const bountyId = publishRequest.bountyId;
 	const guildId = publishRequest.guildId;
+
+	Log.info(`starting to finalize bounty: ${publishRequest.bountyId} from guild: ${publishRequest.guildId}`);
```

Comment:
Minor, but if I had to guess, you moved this line to do: 
```
Log.info(`starting to finalize bounty: ${bountyId} from guild: ${guildId}`);
```

Reply:
Kind of. I originally was going to log a different message but put it back. I fixed it.